toolflow/boot/generate_boot_image.sh

```diff
@@ -1,318 +1,327 @@
 #!/bin/bash
 BOARD=${1:-zedboard}
-VERSION=${2:-2019.1}
-IMGSIZE=${3:-7534}
+VERSION=${2:-2019.2}
+IMGSIZE=${3:-5120}
 SDCARD=${4:-}
 JOBCOUNT=4
 SCRIPTDIR="$(dirname $(readlink -f $0))"
 DIR="$SCRIPTDIR/$BOARD/$VERSION"
 LOGDIR="$DIR/logs"
-CROSS_COMPILE=${CROSS_COMPILE:=arm-linux-gnueabihf-}
-ROOTFS_IMG="$SCRIPTDIR/rootfs.img"
-PYNQ_VERSION="pynq_z1_v2.1"
-PYNQ_IMAGE="$SCRIPTDIR/pynq/$PYNQ_VERSION.zip"
-PYNQ_IMAGE_URL="http://files.digilent.com/Products/PYNQ/$PYNQ_VERSION.img.zip"
+ROOTFS_URL="http://cdimage.ubuntu.com/ubuntu-base/releases/20.04/release/ubuntu-base-20.04-base-armhf.tar.gz"
```

Comment:
Worth using mirrors instead of direct URL?

Reply:
Should not matter for this file size (< 25 MB)

Reply:
Was more a question of: Is this link reliable, or would a mirrorlist or something increase reliability.

Reply:
The main ubuntu server should be reliable enough, would not spend any time on that.